src/bundle/Resources/public/scss/_view.content.scss

```diff
@@ -0,0 +1,14 @@
+.ez-content-view {
+    .ez-content-container {
+        display: flex;
+    }
+
+    .ez-content-tree-container {
+        min-width: 250px;
```

Comment:
Is there any reason not to use `calculateRem(250px)` here?

Reply:
Good catch.